src/main/java/com/softserve/service/impl/UserServiceImpl.java

```diff
@@ -318,4 +318,10 @@ private User saveSocialUser(String email) {
         user.setRole(Role.ROLE_USER);
         return userRepository.save(user);
     }
+
+    @Override
+    public boolean isEmailExist(String email) {
```

Comment:
Why do you need a method that just calls another method?

Reply:
Yes, I missed it. Fixed.